python/tests/test_phylo_formats.py

```diff
@@ -89,22 +89,37 @@ class TestNewick(TreeExamples):
     external Newick parser.
     """
 
-    def verify_newick_topology(self, tree, root=None, node_labels=None):
+    @staticmethod
+    def length_parser(x):
```

Comment:
I don't think we need a static method for this, do we? A static method implies this function is useful outside the immediate test context, whereas this is really just for newick parsing.

 Seems like a good use for a lambda function:
```
 # By default the newick lib outputs length 0.0 if no branch lengths are present
newick_tree = newick.loads(ns, length_parser=lambda x: None if x is None else float(x))[0]
```

Reply:
Happy to use lambda